### ISSUE TITLE
Database is read only when created in app.js

### ISSUE THREAD

**Issue by davidjbeaver** [NathanaelA/nativescript-sqlite#115]
I have an app I am developing that has a mixture of pre shipped and in app created databases.

The shipped databases work fine, and as per other docs I copy this over in the main app.js file.

I just discovered a quirk where if I create the empty database in app.js it appears to be read only when it comes time to insert the records in.  The promise for the execSQL function is never called either.  The DB opens correctly, but the console logs in each promise callback are never called.

I moved the DB creation/table creation into the relevant page back end code and it worked straight away.

It is in a working state, however the main app.js file is logically where you would probably want to initalise the db file if it hasn't yet been created (on first run for example).

I thought it might be related to not calling db.Close() after the insert is completed, however if I leave this off when the code is in the relevant page, subsequent calls still allow me to insert extra rows.

**Comment by NathanaelA** [comment]
1. I would have to see the code; I'm not 100% sure I understand the issue.   
2. What type of device(s) & OS versions

**Comment by davidjbeaver** [comment]
Yes apologies for the poor description.

I need to wait until I get home from work to post the example code.  It was happening on both iOS and Android, iOs using the nativescript preview on a device (running latest iOS version) and Android using an emulator (i think from memory using API 28?? will check).

I am going to run another test closing the DB after creating it in the main app.js entry point when I get home and see if that works, otherwise I will provide a code example

**Comment by davidjbeaver** [comment]
Always happens..

I created a brand new application replicating the situation, and it worked.

I then cleaned the bundle, ran in the emulator and on my iPhone via preview and both times it now worked.

The only thing I can put it down to is the original version I had was created in the online playground, then downloaded where the other 2 were created from the Nativescript Sidekick app.

It's all working fine now though.

Sorry for the confusion!
